images/dcos/Dockerfile

```diff
@@ -1,25 +1,32 @@
-FROM busybox:ubuntu-14.04
+FROM alpine:3.2
```

Comment:
I prefer alpine just because it has a package manager, but I haven't tested DNS resolution.